lib/Dialect/AIE/IR/AIEDialect.cpp

```diff
@@ -735,8 +733,9 @@ int xilinx::AIE::ShimMuxOp::rowIndex() { return getTileOp().rowIndex(); }
 
 // ShimDMAOp
 LogicalResult xilinx::AIE::ShimDMAOp::verify() {
-  assert(getOperation()->getNumRegions() == 1 && "ShimDMAOp has zero region!");
-  assert(!getBody().empty() && "ShimDMAOp should have non-empty body");
+  if (getBody().empty())
+    return emitOpError("cannot have an empty body");
```

Comment:
Sorry, this is the oddball one... All the rest say "should have non-empty body"

Reply:
Same as earlier, I will correct the message.